src/schemas/json/github-funding.json

```diff
@@ -88,6 +88,12 @@
       "description": "Project name on LFX Crowdfunding.",
       "minLength": 1
     },
+    "polar": {
```

Comment:
If there are any restrictions to username, I suggest adding `pattern` property to specify valid username. Do not remove `minLength` as for instance in YAML Red Hat (VS Code extension) it checked before `pattern`: it means at first error about incorrect length is shown, and just when length is correct error about invalid pattern is displayed.

Reply:
Good catch! Since Polar is syncing with GitHub users/organizations, I changed the type to be `github_username`.